assemblyline_v4_service/common/result.py

```diff
@@ -235,9 +235,12 @@ class TextSectionBody(SectionBody):
     def __init__(self, body=None) -> None:
         super().__init__(BODY_FORMAT.TEXT, body=body)
 
-    def add_line(self, text: Union[str, List]) -> None:
+    def add_line(self, text: Union[str, List]) -> Optional[str]:
         # add_line with a list should join without newline seperator.
         # use add_lines if list should be split one element per line.
+        if not text:
```

Comment:
If you pass in an empty string (i.e. you want to print two lines, separated by an empty line), this changes the behaviour of the ResultSection.
You should add a test of
```python
tsb = TextSectionBody()
tsb.add_line("line1")
tsb.add_line("")
assert tsb.add_line("line3") == "line1\n\nline3"
```